src/components/App.tsx

```diff
@@ -81,9 +83,11 @@ const App = (): JSX.Element => {
     RecycledItemsScreen,
     withAuthorizationProps,
   );
+  const MapWithAuthorization = MapItemsScreen;
```

Comment:
So no auth currently ?

Reply:
you can access the map without auth, if the item is public.